tutorials/plugins/gdnative/gdnative-cpp-example.rst

```diff
@@ -61,16 +81,23 @@ to the one described here, as much of the code we'll be showcasing here
 assumes the project follows this layout.
 
 Do make sure you clone recursive to pull in both repositories:
-.. code-block:: none
+
+.. tabs::
+ .. code-tab:: none Godot
+
+    mkdir gdnative_cpp_example
+    cd gdnative_cpp_example
+    git clone --recursive https://github.com/GodotNativeTools/godot-cpp
+
+ .. code-tab:: none Godot 3.0
 
     mkdir gdnative_cpp_example
     cd gdnative_cpp_example
     git clone --recursive -b 3.0 https://github.com/GodotNativeTools/godot-cpp
 
-.. note:: The ``-b 3.0`` I've added as an example to show how to select a specific branch for a specific version of Godot.
-          Also ``godot-cpp`` now includes ``godot_headers`` as a nested submodule, if you've manually downloaded them please make sure to place ``godot_headers`` inside of the ``godot-cpp`` folder.
+.. note:: ``godot-cpp`` now includes ``godot_headers`` as a nested submodule, if you've manually downloaded them please make sure to place ``godot_headers`` inside of the ``godot-cpp`` folder.
 
-          You don't have to do it this way but I've found it easiest to manage. If you decide to just download the repositories or just clone them into your folder, makes sure to keep the folder layout the same as I've setup here as much of the code we'll be showcasing here assumes the project has this layout.
+          You don't have to do it this way but I've found it easiest to manage. If you decide to just download the repositories or just clone them into your folder, make sure to keep the folder layout the same as I've setup here as much of the code we'll be showcasing here assumes the project has this layout.
```

Comment:
It's no big deal, but IMO we should avoid saying "I" in official documentation, as tutorials are not identified as written by a given author, so readers don't know who "I" is. It's probably better to say "we" instead, or avoid the first person altogether if that doesn't impact the style/flow negatively.

Reply:
Given that the whole tutorial is written at the first person ("we"), it's fine to stick to this style. But yeah, avoiding "I" where possible would be nice.